### DISCUSSION TITLE
Link broken in rofi man.

### DISCUSSION THREAD

**Discussion by shivanshu-semwal** [davatorium/rofi#1496] [(Possible) Issues] [answered]
https://github.com/davatorium/rofi/blob/1520a4a5cdd5debd0f829692f9e4147df2bde946/doc/rofi.1.markdown#L648

[https://developer.gnome.org/pygtk/stable/pango-markup-language.html](https://developer.gnome.org/pygtk/stable/pango-markup-language.html) this link is broken for me.

**Comment by sardemff7** [comment, marked as answer]
GNOME changed their developer site some time ago
The new most canonical doc is probably https://docs.gtk.org/Pango/pango_markup.html

**Reply by DaveDavenport** [reply]
updated the link in the manpage.
